onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -273,6 +280,14 @@ def should_quantize(self, node):
         if self.nodes_to_exclude is not None and node.name in self.nodes_to_exclude:
             return False
 
+        # do not quantize non-constant B matrices for matmul
+        if node.op_type == "MatMul":
```

Comment:
why do you need to add this limit? We can specify it from the calling interface

Reply:
@yufenglee Can we specify "if find_by_name(node.input[1], self.model.initializer()) is None:" with the interface? Could you please let me know how to do it? I can remove this by using it. BTW, quantizing non-constant B matrix is always slow from my observation.